mitodl/open-discussions
# When a resource is deleted, delete any UserListItems for that resource

#### Pre-Flight checklist

#### What are the relevant tickets?
Fixes #2387 

#### What's this PR do?
- Ensures that when a resource is deleted, the UserListItems for that resource are also deleted

#### How should this be manually tested?
- Create UserList A
- Create UserList B
- Add UserList B to UserList A
- Delete UserList B
- Go to the detail page for UserList A, it should load and not include UserList B
